cluster/manifests/skipper/pod-deletion-cost-controller.yaml

```diff
@@ -20,6 +20,8 @@ spec:
         application: skipper-ingress
         component: pod-deletion-cost-controller
         deployment: pod-deletion-cost-controller
+      annotations:
+        logging/destination: "{{ .Cluster.ConfigItems.log_destination_infra }}"
```

Comment:
why?

Reply:
We have discussed and agreed to log into both cluster and infra accounts.

Reply:
See #8186